parlai/agents/language_model/language_model.py

```diff
@@ -371,20 +356,22 @@ def batchify(self, observations, seq_len, is_training):
             for obs in observations:
                 if obs:
                     if 'text2vec' in obs:
-                        self.next_batch += obs['text2vec']
-            if len(self.next_batch) <= (seq_len + 1) * self.batchsize:
+                        for vec in obs['text2vec']:
```

Comment:
it's a list of vectors?

Reply:
yes, not sure why i was iterating through it one by one...(i think i must have gotten rid of a check there). i'll fix to just add all at once.